src/index.js

```diff
@@ -21,6 +21,8 @@ export { default as Page } from './ui/Page/Page';
 // export { default as SocialIcon } from './ui/SocialIcon/SocialIcon';
 export { default as Text } from './ui/Text/Text';
 
+import './ui/Errors';
```

Comment:
This should be `export { LoadingError } from './ui/Errors'` or something like that